src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -590,6 +590,7 @@ public void paint(final Graphics g) {
       final AffineTransform t = new AffineTransform();
       t.translate(m_scale * normalizeX(routeDescription.getEnd().getX() - getXOffset()),
           m_scale * normalizeY(routeDescription.getEnd().getY() - getYOffset()));
+      t.translate(mouseShadowImage.getWidth() / -2, mouseShadowImage.getHeight() / -2);
```

Comment:
I'm moving the position of the image when selecting any units to be in the middle of the cursor. Otherwise the upper right corner would be at the cursors position